environment.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - conda-forge
   - default
 dependencies:
-  - python >=3.11,<3.12
+  - python
```

Comment:
Sorry to chime in late, but I would put an upper limit here, or restrict the Python versions to those presently supported, (i.e. `>=3.9,<3.12`). Maybe not necessary, but safer.

Reply:
Also, @aulemahal, you haven't added your authorship here... :eyes:

Reply:
Can't we trust our dependencies and Mamba/Conda to already do that ? numpy/pandas/xarray already follow a strict schedule of python version support.

Reply:
Even using mamba, constricting it a little bit will help speed up the dependency solver. Otherwise, it'll try everything it can.